From 5ec685ab4e495d35ce2ced10f081bf2adf8d4bfe Mon Sep 17 00:00:00 2001
From: Haris Okanovic <haris.okanovic@ni.com>
Date: Wed, 27 Apr 2016 14:35:53 -0500
Subject: [PATCH] config-win32.h: Disable Win32 crypto API on Phar Lap ETS

Don't define USE_WIN32_CRYPTO on Phar Lap ETS. Crypto API in
advapi32.dll is not provided on this platform.

Upstream-Status: Inappropriate [Pharlap specific]
---
 lib/config-win32.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/lib/config-win32.h b/lib/config-win32.h
index fdac6f9a2..4e7ec440e 100644
--- a/lib/config-win32.h
+++ b/lib/config-win32.h
@@ -706,7 +706,9 @@ Vista
 #endif
 
 /* Define to use the Windows crypto library. */
-#define USE_WIN32_CRYPTO
+#if !defined(PHARLAP)
+#  define USE_WIN32_CRYPTO
+#endif
 
 /* Define to use Unix sockets. */
 #if defined(_MSC_VER) && _MSC_VER >= 1900
-- 
2.17.1

